code/index.d.ts

```diff
@@ -1,26 +1,28 @@
-// Type definitions for code 4.0.0
+// Type definitions for code 4.0
 // Project: https://github.com/hapijs/code
 // Definitions by: Prashant Tiwari <https://github.com/prashaantt>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /** Generates an assertion object. */
-export function expect<T>(value: T | T[], prefix?: string): AssertionChain<T>;
+declare function expect<T>(value: T | T[], prefix?: string): AssertionChain<T>;
```

Comment:
```ts
interface Int {};
export function fn(): Int;
```

The above exposes `Int` along with `fn` to the consumer which is internal implementation of the type definition and not part of the original module's API.

```ts
interface Int {};
declare function fn(): Int;
export { fn };
```

This seems to solve the problem: `Int` is now self-contained and not available to consumers for importing.

Is there a better way to handle this?